doc/user/content/sql/isolation-level.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Isolation Level"
+description: "The isolation level defines how Materialize isolates the execution of transactions."
+aliases:
+ - /sql/consistency
+menu:
+  main:
+    parent: 'reference'
+    weight: 160
+---
+
+The SQL standard defines four levels of transaction isolation. In order of least strict to most strict they are Read
+Uncommitted, Read Committed, Repeatable Read, and Serializable. In Materialize, you can request any of these isolation
```

Comment:
This could be styled as:

```
...most strict they are:

* Read Uncommitted
* Read Committed
* Repeatable Read
* Serializable
```

In Materialize...

Reply:
Good idea, I just updated it.